test/eni/IPRelease.go

```diff
@@ -0,0 +1,180 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2020-2021 Authors of Cilium
```

Comment:
nit: We're removing the copyright date, so `Copyright Authors of Cilium` is sufficient here

https://github.com/cilium/cilium/pull/18813/files

Reply:
Fixed it.